addons/website/static/src/snippets/s_share/000.js

```diff
@@ -4,18 +4,24 @@ import publicWidget from 'web.public.widget';
 
 const ShareWidget = publicWidget.Widget.extend({
     selector: '.s_share, .oe_share', // oe_share for compatibility
-    events: {
+    read_events: {
         'click a': '_onShareLinkClick',
     },
+    disabledInEditableMode: false,
 
     /**
      * @override
      */
     async start() {
-        this.URL_REGEX = /(\?(?:|.*&)(?:u|url|body)=)(.*?)(&|#|$)/;
-        this.TITLE_REGEX = /(\?(?:|.*&)(?:title|text|subject|description)=)(.*?)(&|#|$)/;
-        this.MEDIA_REGEX = /(\?(?:|.*&)(?:media)=)(.*?)(&|#|$)/;
-
+        if (!this.editableMode) {
+            this.URL_REGEX = /(\?(?:|.*&)(?:u|url|body)=)(.*?)(&|#|$)/;
+            this.TITLE_REGEX = /(\?(?:|.*&)(?:title|text|subject|description)=)(.*?)(&|#|$)/;
+            this.MEDIA_REGEX = /(\?(?:|.*&)(?:media)=)(.*?)(&|#|$)/;
+        }
+        // TODO: adapt in master. This will disable the link popover on "s_share" items
+        // in stable versions. It can be replaced simply by adding "o_no_link_popover"
+        // class in XML.
+        this.el.classList.add('o_no_link_popover');
```

Comment:
Don't add this via the widget. Can't you do it via an edit-mode only option? At worst, on start of the edit mode as a temporary patch?

Reply:
Actually, I moved the fix code to the widget because I could not use any `data-js` in stable to target the snippet...
I suggest to override the `weSnippetEditor.callPostSnippetDrop()` on website `include` (current PR code), this should have the same impact "on stable" as using `onBuilt` in options  (since we need to disable link popover right after the snippet is dropped into the DOM in this case :+1:). What do you think?

Reply:
Yes you can do something like that :+1: As long as it is commented etc, and adapted in the master forward-ported PR as always :+1:

Reply:
Yes, the PR is already adapted with the new code, and splitted into two commits, just let me know if something needs to be updated. Thanks :+1: